templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+**Feature level 133**
+
+* [`PATCH /streams/{stream_id}`](/api/update-stream): The endpoint now
+  returns a 404 response for a request to make a public stream with
+  protected history which was previously ignored silently.
```

Comment:
Not sure where to add `Changes` entry for this in `zulip.yaml`.

Reply:
We could add it to the end of the endpoint's main description. And/or we could also add an example of the 404 error response with the `Changes` entry in the description there.

Reply:
I think in the `history_public_to_subscribers` description would be a good place.

In fact I think the old behavior is already accurately described by the "Changes" entry that the next commit adds there, except for the part where it says what feature level the change happened at. (That might be reason enough to squash these two commits together -- this one is small, and logically goes with that later commit anyway.)

It'd be good to also describe the new semantics, outside the "Changes" entry: it's an error for this parameter to be false when is_private is false.

Reply:
> except for the part where it says what feature level the change happened at.

Oh I see, and in fact this commit doesn't bump the feature level.

Yeah, better to squash these, I think.

Reply:
I have added in the description that setting the parameter false when is_private is false or when stream is public/web-public will raise an error.
And squashed the last two commits.